src/consensus/pbft/pbft.h

```diff
@@ -211,79 +207,84 @@ namespace pbft
 
       message_receiver_base->register_append_ledger_entry_cb(
         append_ledger_entry_cb, ledger.get());
+    }
 
-      on_request = [&](kv::TxHistory::RequestCallbackArgs args) {
-        auto total_req_size = pbft_config->message_size() + args.request.size();
-
-        uint8_t request_buffer[total_req_size];
-        pbft_config->fill_request(
-          request_buffer,
-          total_req_size,
-          args.request,
-          args.actor,
-          args.caller_id);
-
-        auto rep_cb = [&](
-                        void* owner,
-                        kv::TxHistory::RequestID caller_rid,
-                        int status,
-                        uint8_t* reply,
-                        size_t len) {
-          LOG_DEBUG_FMT("PBFT reply callback for {}", caller_rid);
-
-          return rpcsessions.reply_async(
-            std::get<1>(caller_rid), {reply, reply + len});
-        };
-
-        LOG_DEBUG_FMT("PBFT sending request {}", args.rid);
-        return client_proxy->send_request(
-          args.rid,
-          request_buffer,
-          sizeof(request_buffer),
-          rep_cb,
-          client_proxy.get());
+    bool on_request(const kv::TxHistory::RequestCallbackArgs& args) override
+    {
+      LOG_INFO_FMT("in on request of pbft");
```

Comment:
in on_request?

Reply:
this debug, or more likely trace, not info.

Reply:
thanks' forgot that there